asmcli/components/ca/managed-cas.sh

```diff
@@ -75,6 +75,54 @@ x_enable_workload_certificate_on_membership() {
      -d "${ENABLEFEATURE}" "https://${GKEHUB_API}/v1alpha/projects/${FLEET_ID}/locations/global/features/workloadcertificate?update_mask=membership_specs"
 }
 
+x_wait_for_enabling_workload_certificates() {
+  local GKEHUB_API; GKEHUB_API="$1"
+  local FLEET_ID; FLEET_ID="$2"
+
+  info "Waiting for the workload certificates enablement. This may take up to 20 minutes ..."
+  # Enabling workload certificate feature on a cluster will result in
+  # the cluster restarting and the cluster becoming unreachable for minutes.
+  # The cluster restart may happen at a random time after the enablement.
+  # Therefore, wait 15 minutes before checking the connection to k8s is recovered.
+  sleep 900
```

Comment:
IMO, this i a long wait time..should we perhaps try a smaller initial wait followed by a larger number of retries?

Reply:
A smaller initial wait followed by a larger number of retries does not always work because the cluster restart may happen at a random time after the enablement. For example, if the initial wait is 1 minute, the cluster may not restart yet and thus passing the check. Then after 1 minute, the cluster begins restarting and causes the installation to fail.